src/Client.php

```diff
@@ -242,7 +242,7 @@ public function authenticate($code)
      * @param string $code code from accounts.google.com
      * @return array access token
      */
-    public function fetchAccessTokenWithAuthCode($code)
+    public function fetchAccessTokenWithAuthCode($code, $codeVerifier = null)
```

Comment:
Should the codeVerifier parameter be added to the doc comment above?

Reply:
Good catch, done!